js/coinsbit.js

```diff
@@ -0,0 +1,1114 @@
+        if (since !== undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrders does not have an usable since parameter, so please unset it');
+        }
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        const offset = this.safeValue (params, 'offset');
+        if (offset !== undefined) {
+            request['offset'] = offset;
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privatePostOrders (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //         limit: '50',
+        //         offset: '0',
+        //         total: '1',
+        //         result: [
+        //            {
+        //               id: '8633074431',
+        //               left: '0.002',
+        //               market: 'ETH_USDT',
+        //               amount: '0.002',
+        //               type: 'limit',
+        //               price: '2000.1892',
+        //               timestamp: '1640284932.136',
+        //               side: 'buy',
+        //               dealFee: '0',
+        //               takerFee: '0.002',
+        //               makerFee: '0.002',
+        //               dealStock: '0',
+        //               dealMoney: '0'
+        //           },
+        //           ...
+        //         ]
+        //     },
+        //     code: '200'
+        // }
+        const data = this.safeValue (response, 'result', []);
+        const orders = this.safeValue (data, 'result', []);
+        return this.parseOrders (orders, market, since, limit);
+    }
+
+    async fetchClosedOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        request['orderId'] = id;
+        const response = await this.privatePostAccountOrder (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       offset: '0',
+        //       limit: '50',
+        //       records: {
+        //         time: '1640258718.402',
+        //         fee: '0.01572657',
+        //         price: '3931.64271609',
+        //         amount: '0.002',
+        //         id: '1595921',
+        //         dealOrderId: '8631729321',
+        //         role: '2',
+        //         deal: '7.86328543'
+        //       }
+        //     },
+        //     code: '200'
+        // }
+        const result = this.safeValue (response, 'result', {});
+        const order = this.safeValue (result, 'records');
+        if (order === undefined) {
+            throw new OrderNotFound (this.id + ' fetchOrder() could not find any closed order data for id ' + id);
+        }
+        return this.parseMyTrade ({ 'id': id, 'order': order }, undefined);
```

Comment:
We can't use a trade parser to parse an order, something must be wrong here. It's either fetchClosedOrder that must be renamed or it's parseMyTrade.

Reply:
Also, this is the only mention of parseMyTrade throughout this file, so it can't be right.

Reply:
This is the last fix as I see and I am doing that now. Will be pushed shortly.